.travis.yml

```diff
@@ -4,22 +4,23 @@ services:
 script:
 - docker build --target test --tag my-test-image .
 - docker run my-test-image todo_app/items_test.py    
-- docker run -e TOKEN -e KEY -e TOID -e DOINGID -e DONE -e CLIENT -e GITHUBID -e WEBAPPLICATIONCLIENT -e CLIENTSECRET -e TODOBOARD -e DOINGBOARD -e DONEBOARD -e CONNECTIONSTRING -e WEBHOOK_URL my-test-image  todo_app/e2e_test.py 
+- docker run -e TOKEN -e KEY -e CLIENT -e GITHUBID -e WEBAPPLICATIONCLIENT -e CLIENTSECRET -e TODOBOARD -e DOINGBOARD -e DONEBOARD -e CONNECTIONSTRING my-test-image  todo_app/e2e_test.py 
 - docker build --target production --tag my-production-image .
-- bash webhook.sh
+
 
 before_deploy:
 - echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
-- docker build --target production --tag $DOCKER_USERNAME/my-production-image:${TRAVIS_COMMIT::8} .
-- docker push $DOCKER_USERNAME/my-production-image:${TRAVIS_COMMIT::8}
+- docker build --target production --tag $DOCKER_USERNAME/my-production-image:latest .
+- docker push $DOCKER_USERNAME/my-production-image:latest
 - docker login --username="$DOCKER_USERNAME" --password=$(heroku auth:token) registry.heroku.com
-- docker pull $DOCKER_USERNAME/my-production-image:${TRAVIS_COMMIT::8}
-- docker tag  $DOCKER_USERNAME/my-production-image:${TRAVIS_COMMIT::8} registry.heroku.com/trello-to-do/web
-- docker push registry.heroku.com/trello-to-do/web
+- docker pull $DOCKER_USERNAME/my-production-image:latest
+- docker tag  $DOCKER_USERNAME/my-production-image:latest registry.heroku.com/trello-to-do/web
+# - docker push registry.heroku.com/trello-to-do/web
```

Comment:
Given you're not pushing to the Heroku registry any more, we might as well remove the relevant tagging step. And we should remove these rather than just comment them out - we can always recover them with Git if necessary!